portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,4 +29,4 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
   }
-}
+}
```

Comment:
nit: Since this file has no real changes, I would remove it from the changes in the pull request for cleanliness. It's not necessary, but just a minor nit pick. Sometimes IDEs add or remove line endings at the end of touched files and that can add noise to the file history.

Reply:
It might also be good practice adding or removing files from a pull request (PR) or having files that have changed but are not included in a commit/PR.

Reply:
Ok, noted!